src/assets/main.scss

```diff
@@ -40,18 +45,21 @@ $active-node-color: #0076d6;
         width: 100%;
         display: flex;
         overflow:hidden;
-        > * {
+        padding: .5rem .75rem;
+        > .title {
             flex-grow: 1;
-            display: block;
-            padding: .5rem .75rem;
+            display: -webkit-box;
```

Comment:
Any reason for using a Chrome-specific extension? And does this behave as expected on non-Chrome browsers?

Reply:
used this extension as it provided a simple way for line clamping.

<img width="492" alt="image" src="https://github.com/sg-developer-portal/docsify-sidebar-collapse/assets/35069055/6cdcf773-f1b5-47dc-b937-944635ac397c">

Expected to work fine for most of the main browsers except (IE). 
Webkit box was built for safari and chrome. Microsoft edge uses chromium and tested locally for firefox as well to make sure it works.